clients/cli/src/main/java/org/apache/gravitino/cli/Main.java

```diff
@@ -35,6 +35,11 @@ public static void main(String[] args) {
     try {
       CommandLine line = parser.parse(options, args);
       String entity = resolveEntity(line);
+      String[] extra = line.getArgs();
+      if (extra.length > 2) {
```

Comment:
One command can only has 1 argument?

Reply:
the arguments don't include options and all commands have a maximum of two arguments.